permissions/component-file-leak-detector.yml

```
@@ -0,0 +1,7 @@
+---
+name: "file-leak-detector"
+github: "jenkinsci/file-leak-detector"
```

```suggestion
github: "jenkinsci/lib-file-leak-detector"
```

@timja Feel free to rename this to jenkinsci/lib-file-leak-detector if you want it to be consistent with the other libraries we host.